README.md

```diff
@@ -41,7 +42,6 @@ Instantiate the environment and run the tests:
 (IceFloeTracker) pkg> test
 ```
 
-
 ## Notebooks
```

Comment:
We do have some Jupyter notebooks though, I added a statement just alerting users that they are available, but it would be up to the user to figure out how run a `.ipynb`.